### PR TITLE
AGENTOPS-25264: Deploy te-browserbot.yaml on cloud agents

### PR DESCRIPTION
**Pull request by brunoofe-te**
https://thousandeyes.atlassian.net/browse/AGENTOPS-25264

### CODE REVIEW

File: manifests/config.pp [hunk 1]
```diff
@@ -9,8 +9,13 @@
 class te_agent::config {
 
   $te_agent_file_ensure = $te_agent::agent ? {
-    true  => present,
-    false => absent,
+    true  => 'present',
```
Comment (lalgarvi-te) [inline manifests/config.pp:12]: we are not supposed to quote states in resources,
however in variables is a gray area, im not sure that applies
https://www.puppet.com/docs/puppet/8/style_guide.html#quoting

**Review by lalgarvi-te — APPROVED** [review]

**Comment by brunoofe-te** [comment]
I will test a different approach before changing this to "ready for review"

**Review by avarnell-te — APPROVED** [review]
Looks good from a bbot side, but I am not well versed in pp files or what puppet does under the hood

**Comment by brunoofe-te** [comment]
another revamp... the final one

**Comment by brunoofe-te** [comment]
added a service for BBOT so it can be restarted when the configuration changes.
final code... ready for review

**Review by lalgarvi-te — APPROVED** [review]

**Pull request merged**